src/main/java/org/folio/consortia/controller/SharingInstanceController.java

```diff
@@ -32,4 +33,11 @@ public ResponseEntity<SharingInstance> start(UUID consortiumId, @Validated Shari
   public ResponseEntity<SharingInstance> getSharingInstanceById(UUID consortiumId, UUID actionId) {
     return ResponseEntity.ok(sharingInstanceService.getById(consortiumId, actionId));
   }
+
+  @Override
+  public ResponseEntity<SharingInstanceCollection> getSharingInstances(UUID consortiumId, UUID instanceIdentifier,
+      String sourceTenantId, String targetTenantId, String status, Integer offset, Integer limit) {
```

Comment:
Lets use enum for 'status' here instead of String, also lets add allowed values to the corresponding yaml file

Reply:
'status' is defined as String in the [schema ](https://github.com/folio-org/mod-consortia/blob/e2a1d18ce4d003c74dca48177e6f52d40ba20e1a/src/main/resources/swagger.api/schemas/sharingInstance.yaml#L14)of _sharingInstance_ . So chose String here, but we have [conversion](https://github.com/folio-org/mod-consortia/pull/58/files#diff-38a6864c6c8cfe87fb3e89896e6783325713c2fc5867f2971a2c1dc14378e779R43:~:text=SharingInstanceEntity.StatusType.valueOf(status)), and for invalid values we get 400.

Reply:
Thank you, I have created _[Status](https://github.com/folio-org/mod-consortia/blob/3bad07ce54a899f8d9cb1515653e581f99e1e296/src/main/resources/swagger.api/schemas/status.yaml#L1)_ enum and used it in all places.